ports/krabsetw/vcpkg.json

```
@@ -0,0 +1,7 @@
+{
+  "name": "krabsetw",
+  "version-string": "4.1.13",
+  "description": "krabsetw is a C++ library that simplifies interacting with ETW.",
```

```suggestion
  "description": "krabsetw is a header-only C++ library that simplifies interacting with ETW.",
```
It would be nice to add header-only here.